modules/openapi-generator/src/main/resources/kotlin-spring/beanValidationQueryParams.mustache

```diff
@@ -0,0 +1 @@
+{{#required}}@NotNull {{/required}}{{>beanValidationCore}}
```

Comment:
`beanValidationCore` template is missing in this PR, and causes generation to fail.